soc/xtensa/intel_adsp/common/include/intel-adsp-ipc-regs.h

```diff
@@ -0,0 +1,94 @@
+/* Copyright (c) 2022 Intel Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_SOC_INTEL_ADSP_IPC_REGS_H
+#define ZEPHYR_SOC_INTEL_ADSP_IPC_REGS_H
+
+#include <zephyr/devicetree.h>
+#include <stdint.h>
+
+/**
+ * @brief IPC registers layout for every Intel ADSP SoC family.
+ */
+struct intel_adsp_ipc {
+#if defined(CONFIG_INTEL_ADSP_CAVS)
+#if defined(CONFIG_SOC_SERIES_INTEL_CAVS_V15)
+	uint32_t tdr;
+	uint32_t tdd;
+	uint32_t idr;
+	uint32_t idd;
+	uint32_t ctl;
+	uint32_t ida; /* Fakes for source compatibility; these ...  */
+	uint32_t tda; /* ... two registers don't exist in hardware. */
+#else /* Remaining CAVS SoCs */
+	uint32_t tdr;
+	uint32_t tda;
+	uint32_t tdd;
+	uint32_t unused0;
+	uint32_t idr;
+	uint32_t ida;
+	uint32_t idd;
+	uint32_t unused1;
+	uint32_t cst;
+	uint32_t csr;
+	uint32_t ctl;
+#endif /* CONFIG_SOC_SERIES_INTEL_CAVS_V15 */
+#endif /* CONFIG_INTEL_ADSP_CAVS */
+
+#ifdef CONFIG_SOC_SERIES_INTEL_ACE1X
+	uint32_t tdr;
+	uint32_t tda;
+	uint32_t unused0[2];
+	uint32_t idr;
+	uint32_t ida;
+	uint32_t unused1[2];
+	uint32_t cst;
+	uint32_t csr;
+	uint32_t ctl;
+	uint32_t cap;
+	uint32_t unused2[52];
+	uint32_t tdd;
+	uint32_t unused3[31];
+	uint32_t idd;
+#endif /* CONFIG_SOC_SERIES_INTEL_ACE1X */
+};
+
+#define INTEL_ADSP_IPC_BUSY BIT(31)
+#define INTEL_ADSP_IPC_DONE BIT(31)
```

Comment:
Some bit for both?

Reply:
Yes it is. It's just a rename of:
![image](https://user-images.githubusercontent.com/12482515/178689906-1ef530f7-7dbe-4bb9-b360-5c0e02852de2.png)
IT's just written to different registers:
BUSY to TDR and IDR
DONE to TDA and IDA